x-pack/plugins/stack_alerts/server/rule_types/es_query/rule_type.ts

```diff
@@ -135,6 +142,20 @@ export function getRuleType(
     }
   );
 
+  const alerts: IRuleTypeAlerts<StackAlert> = {
+    context: STACK_AAD_INDEX_NAME,
+    mappings: {
+      fieldMap: {
+        [ALERT_URL]: { type: 'text', array: false, required: false },
+        [ALERT_TITLE]: { type: 'keyword', array: false, required: false },
+        [ALERT_REASON]: { type: 'keyword', array: false, required: false },
```

Comment:
REASON  should be in the framework fields so we shouldn't need to redefine it here